app/src/main/res/values/strings.xml

```diff
@@ -1432,6 +1432,8 @@
     <string name="title_ongoing_notification_channel">Use ongoing notification channel</string>
     <string name="summary_notification_channels_grouping">Whether notifications should be grouped together. Incompatible with \'Number Item in Notification Area\' option</string>
     <string name="title_notification_channels_grouping">Group notifications together</string>
+    <string name="summary_delay_ascending_3min">For ascending volume profile, delay the start of the alert sound by 3 minutes</string>
```

Comment:
I would write here something like:
"For ascending volume profile, only vibrates for the first 3 minutes"

Reply:
Thanks for the review.

I don't understand your comment.
I just tested xDrip.  It doesn't vibrate just for the first 3 minutes.  It continues to vibrate until the alert is snoozed.

This PR does not make any changes to the vibration behavior.
The only change this PR makes is with the sound behavior, and only if the new setting is disabled.

Would you please explain what you mean by your suggestion?

Thanks again

Reply:
May be I'm starting to understand your point.  Do you mean this summary says nothing about the vibration and the user may think that the vibration is also delayed?

If yes, what if I change the summary to the following?

For ascending volume profile, delay the start of the alert sound by 3 minutes.  The vibration (if the vibration is enabled) starts as soon as the alert triggers regardless of this setting.

Reply:
yes, your last suggestion seems what I meant.